warehouse/accounts/models.py

```diff
@@ -163,6 +166,17 @@ def recent_events(self):
             .all()
         )
 
+    @property
+    def password_reset_prohibited(self):
```

Comment:
It's minor, but from my experience, things tend to get clearer when booleans are expressed through something positive, not negative (e.g. `can_reset_password`) otherwise it tends to involve double-negatives.

Reply:
👍🏼 similar feedback from someone else so I'll refactor.